operations/messages.md

```diff
@@ -45,13 +50,15 @@ Get all messages belonging to the specified [Message threads](messagethreads.md#
             },
             "CreatedUtc": "2022-03-09T13:19:46Z"
         }
-    ]
+    ],
+    "Cursor": "8d02142f-31cf-4115-90bf-ae5200c7a1ba"
 }
 ```
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Messages` | array of [Messages](#message) | required | The filtered messages. |
+| `Cursor` | string | required | Index of the last message Id. Should be used in [Limitation](messagethreads.md#limitation) to fetch older messages in the follow-up request. |

```

Comment:
The required is not correct here no? If you will use the wrong time filter + limitation and there will be no item in the response, there will be no cursor provided. Or am I missing something?

Reply:
yeah, although cursor Field might be there, the value will be null -> optional.. I'm changing it.